ci/output-results-to-slack.yaml

```diff
@@ -3,7 +3,7 @@ image_resource:
   type: docker-image
   source:
     repository: google/cloud-sdk
-    tag: slim
+    tag: latest
```

Comment:
You should be able to use:
```
    repository: python
    tag: 3.8
```
It is a smaller image than gcloud-sdk latest plus we don't actually use need any of the cloud-sdk cli commands.